scripts/load-inline

```diff
@@ -0,0 +1,14 @@
+Cities:
+  LOAD * Inline 
```

Comment:
Do we think of `Inline` as a reserved word in the script syntax?
If yes, maybe we should keep all such reserved word printed in uppercase (`INLINE`)?